hazelcast/src/main/java/com/hazelcast/map/impl/MapSplitBrainHandlerService.java

```diff
@@ -60,6 +60,9 @@ protected void onStoreCollection(RecordStore recordStore) {
         DefaultRecordStore defaultRecordStore = (DefaultRecordStore) recordStore;
         defaultRecordStore.getMapDataStore().reset();
         defaultRecordStore.getIndexingObserver().onDestroy(false, true);
+
+        MapContainer mapContainer = defaultRecordStore.getMapContainer();
```

Comment:
Could you add a comment describing the reasoning of the removal this map container in this sync call? 
I could understand for now because I knew the pr's context, issue that it's solved etc., but it'd be really hard to understand by just only looking at the code.

Reply:
done.